### ISSUE TITLE
certbot-auto apache config parse failure with greater than sign

### ISSUE THREAD

**Issue by v12mike** [certbot/certbot#3536]
> @hokascha I've filed #1724 to track the issue you're experiencing, but I don't have a fix for it at the moment apart from editing the config to add the missing quote.
> 
> If anybody else has Apache config file parse failures, please file a new ticket at https://github.com/letsencrypt/letsencrypt/issues/new to avoid more confusion, and feel free to mention @domcleal so I can try and help.

I stumbled across another parse error,  when my apache config contains the snippet below:

It is not a problem for me, as I can delete the whole snippet, but as it is (I think) syntactically correct, I thought you might like to patch your app to accept it.

```
#   send 410 page gone response for non-existent pages to avoid cluttering error logs
    RewriteMap movedPageMap txt:conf/movedPage.map
    RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} !-f
    RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} !-d
    RewriteCond ${movedPageMap:$1}  >""
    RewriteRule ^.*$ /movedPage.php [L]
```

**Comment by bmw** [comment]
cc @domcleal

**Comment by bmw** [comment]
I believe this issue was fixed in version 0.28.0 of the Apache plugin. If anyone is still hitting this issue, please comment or open a new issue with the relevant lines in your Apache config.
